src/run/parser/wsplit.rs

```diff
@@ -97,7 +97,7 @@ pub fn parse<R: BufRead>(source: R, load_icons: bool) -> Result<Run> {
     }
 
     if old_run_exists {
-        run.add_custom_comparison("Old Run");
+        run.add_custom_comparison("Old Run").ok();
```

Comment:
same as above

Reply:
actually possibly not, as "Old Run" should always be a valid comparison. So it should probably just be .expect(...)